CONTRIBUTING.md

```diff
@@ -1,30 +1,151 @@
 # Contributing to `@lmc-eu/spirit-design-system`
 
-First of all, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
+First of all, thanks for your contribution to this project! ❤️
+Here are some tips how to make your contributing efforts efficient and eventually accepted & merged:
 
-## General usage
+- [General Usage](#gu)
+- [Project structure](#ps)
+- [Commit messages](#cm)
+- [Code style](#cs)
+- [Publishing](#pub)
+
+## <a name="gu"></a> General usage
 
 This project uses `makefile` for managing various chores, like dependency installation, testing, linting etc. Make sure you run `make` right after you clone the repository - it will set you up with everything needed to get started. Once you are done with that, have a look at what commands (targets) are available for you to run using `make` - `make install` etc.
 
-## Project structure
+## <a name="ps"></a> Project structure
 
 This project is a monorepo managed by [Lerna][lerna-home]. This means that each folder inside the [packages/](../packages) directory represents a package published to npm. The root directory also contains _package.json_ file but this is only used for local development purposes and does not represent something that is ever published to npm (you might notice that it only contains _devDependencies_ and no prod deps 🤷‍♂️).
 
-## Commit messages
+## <a name="cm"></a> Commit messages
 
 All commits you make should adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with slight modification of our own - [@lmc-eu/commitlint-config][commitlint-config]. This is later used during release phase to determine how to bump the packages' version numbers based on commit history. 🚀
 
-Message of commit must contain specific phrases, like `<type>(<package>): <commit-message> #<issue-id>` eg.`Feat(web-react): React Modal implementation #DS-331`
+### Commit Message Format
+
+We have very precise rules over how our Git commit messages must be formatted.
+This format leads to **easier to read commit history**.
+
+Each commit message consists of a **header**, a **body**, and a **footer**.
+
+```
+<header>
+<BLANK LINE>
+<body>
+<BLANK LINE>
```

Comment:
It will be treated as it is. Conventional changelog parses only certain lines like the header. So this is only convention on how to make commit messages more readable by separating the commit itself into three parts. Blank lines have no impact on how do commit will be parsed.